src/main/resources/External_Resources.csv

```diff
@@ -0,0 +1,115 @@
+UniProt Update,uniprot_xml2sql_isoform.pl,UniprotUpdate.pm#L33,uniprot-reviewed:no.list.gz,TrEMBL entries from UniProt,File,https://www.uniprot.org/uniprot/?query=reviewed%3Ano&compress=yes&format=list
+UniProt Update,uniprot_xml2sql_isoform.pl#L293,EnsEMBLUtils.pm#L70,/lookup/id/[gene_id],EnsEMBL REST endpoint to query if gene is canonical,REST EndPoint,https://rest.ensembl.org/lookup/id/ENSG00000157764
+GO Update,go_obo_update.pl,go_obo_update.pl#L25,go.obo,GO term entries flat file from Gene Ontology,File,http://current.geneontology.org/ontology/go.obo
+GO Update,go_obo_update.pl,go_obo_update.pl#L28,ec2go,Mapping of EC numbers to GO terms,File,http://geneontology.org/external2go/ec2go
+GO Update,go_obo_update.pl,go_obo_update.pl#L445,EBI GO Term page,Quick GO summary page for GO term (linked to in Wiki report),Web Page,https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000001
+ChEBI Update,data-release-pipeline/chebiupdate/Main.java,data-release-pipeline/chebiupdate/ChebiDataRetriever.java#L206,ChEBI Web Services,Source code and description for accessing ChEBI entries,Web Page,https://www.ebi.ac.uk/chebi/webServices.do#Java
+Orthopair,data-release-pipeline/orthopairs/Main.java,data-release-pipeline/orthopairs/Main.java#L80,QfO_Genome_Orthologs.tar.gz,Ortholog data from Panther,File,ftp://ftp.pantherdb.org/ortholog/current_release/QfO_Genome_Orthologs.tar.gz
+Orthopair,data-release-pipeline/orthopairs/Main.java,data-release-pipeline/orthopairs/Main.java#L80,Orthologs_HCOP.tar.gz,Ortholog data from Panther (needed for species including sscr and others),File,ftp://ftp.pantherdb.org/ortholog/current_release/Orthologs_HCOP.tar.gz
+Orthopair,data-release-pipeline/orthopairs/Main.java,data-release-pipeline/orthopairs/Main.java#L96,HGNC_homologene.rpt,Mouse gene to protein mapping file,File,http://www.informatics.jax.org/downloads/reports/HGNC_homologene.rpt
+Orthopair,data-release-pipeline/orthopairs/Main.java,data-release-pipeline/orthopairs/Main.java#L96,GENES_RAT.txt,Rat gene to protein mapping file,File,ftp://ftp.rgd.mcw.edu/pub/data_release/GENES_RAT.txt
+Orthopair,data-release-pipeline/orthopairs/Main.java,data-release-pipeline/orthopairs/Main.java#L96,GenePageEnsemblModelMapping.txt,Frog gene to protein mapping file,File,ftp://ftp.xenbase.org/pub/GenePageReports/GenePageEnsemblModelMapping.txt
+Orthopair,data-release-pipeline/orthopairs/Main.java,data-release-pipeline/orthopairs/Main.java#L96,ensembl_1_to_1.txt,Zebra Fish gene to protein mapping file,File,https://zfin.org/downloads/ensembl_1_to_1.txt
+AddLinks,add-links/Main.java,add-links/src/main/resources/basic-file-retrievers.xml#L27,genes_diseases_external_references.xml,Orphanet Data (PASSWORD REQUIRED),File,http://www.orphadata.org/data/BDD_partners/genes_diseases_external_references.xml

```

Comment:
Um, are these line numbers entered _manually_? Will this file need to be updated if the basic-file-retrievers.xml file is modified and the line numbers change???

Reply:
Yes, I did determine it manually.  It's not critical information and it doesn't have to be included, but it is nice to know where exactly in the source it is.  I can change it to be a file name and leave it at that unless we have a convenient way to update it.

Reply:
I think keeping the line numbers in is nice for when things fail and you want a report, but also is a potential maintenance nightmare if the line numbers here get out of sync with the actual files.

Reply:
Yeah, I agree.  I'll give this more thought.

Reply:
Created issue #3